docs/source/azure/policy/genericarmaction.rst

```diff
@@ -84,3 +84,40 @@ Tags
 
       .. c7n-schema:: TagTrim
             :module: c7n_azure.actions
+
+``DeleteAction``
+      Perform delete operation on any ARM resource. Can be used with 
+      generic resource type `armresource` or on any other more specific
+      ARM resource type supported by Cloud Custodian.
+
+      .. code-block:: yaml
+
+          - policies:
+              - name: delete-test-resources
+                description: |
+                  Deletes any ARM resource with 'test' in the name
+                resource: azure.armresource
+                filters:
+                 - type: value
+                   name: test
+                   op: in
+                actions:
+                 - type: delete
+
+
+        .. code-block:: yaml
+
+          - policies:
+              - name: delete-test-nsg
+                description: |
+                  Deletes any Network Security Group with 'test' in the name
+                resource: azure.networksecuritygroup
+                filters:
+                 - type: value
+                   name: test
+                   op: in
+                actions:
+                 - type: delete
+
+      .. c7n-schema:: DeleteAction
```

Comment:
I see you copied the way tag trim already was, but do you mind just fixing both of them.  Generally in docs the schema would be shown right after the name/description rather than down at the bottom under examples, since it is the part that tells you how to use it.  If you can just move the schemas above the policy examples and under the description it will keep us from having to come back and clean this up later.